### PR TITLE
first cut of conda yaml

### PR DESCRIPTION
**Pull request by virgesmith**

### CODE REVIEW

File: recipes/ukcensusapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "ukcensusapi" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a4/88/e13b9136723fb4617041904243cd24ccddcff5324868c963ab72564d04bc/{{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/ukcensusapi/meta.yaml]: Can you use the canonical url: 

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

File: recipes/ukpopulation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{% set name = "ukpopulation" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/06/4e/bd7d79d83a0b5535dabfec2bc245bec1a30cf918c4dd2bba336a3c4d1a7c/{{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/ukpopulation/meta.yaml]: Use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

File: recipes/ukpopulation/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - ukcensusapi
+
+test:
+  imports:
+    - ukpopulation.myedata
+    - ukpopulation.snppdata
+    - ukpopulation.nppdata
+
+about:
+  home: https://github.com/nismod/{{ name }}
```
Comment (synapticarbors) [inline recipes/ukpopulation/meta.yaml]: Please don't use jinja template variables in the urls in the `about` section.

File: recipes/ukpopulation/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: LICENCE
+  summary: Download, cache, collate, filter and extrapolate UK Population estimates and projections
+  dev_url: https://github.com/nismod/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - virgesmith
+    - tomalrussell
```
Comment (synapticarbors) [inline recipes/ukpopulation/meta.yaml]: @tomalrussell -- are you ok being added as a co-maintainer of the `ukpopulation` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role

Comment (tomalrussell) [reply recipes/ukpopulation/meta.yaml]: Yes, that's fine.

**Comment by synapticarbors** [comment]
@virgesmith -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**